examples/medical-center/generated/persist_client.bal

```diff
@@ -29,7 +29,7 @@ public client class MedicalCenterClient {
             },
             keyFields: ["itemId"]
         },
-        "medicalneed": {
+        "medicalneeds": {
```

Comment:
```suggestion
        [MEDICAL_NEED]: {
```

Reply:
Fixed in [d0b65f0](https://github.com/ballerina-platform/module-ballerina-persist/pull/177/commits/d0b65f00336f1917043ae3b82e35b0c405656d9f)